wagtail/admin/templates/wagtailadmin/home.html

```diff
@@ -30,3 +31,27 @@ <h1>{% block branding_welcome %}{% blocktrans %}Welcome to the {{ site_name }} W
         <p>{% trans "This is your dashboard on which helpful information about content you've created will be displayed." %}</p>
     {% endif %}
 {% endblock %}
+
+{% block extra_js %}
+<script>
+$(function() {
+    $('.object.collapsible').each(function() {
+        var $target = $(this);
+        var $content = $target.find('.object-layout');
+        if ($target.hasClass('collapsed') && $target.find('.error-message').length == 0) {
+            $content.hide();
+        }
+
+        $target.find('> .title-wrapper').on('click', function() {
+            if (!$target.hasClass('collapsed')) {
+                $target.addClass('collapsed');
+                $content.hide('slow');
```

Comment:
Should this be "show"? Also, the collapse/expand icon isn't focusable, so from a brief look I think this is likely to be inaccessible to screenreader users. Collapsing/expanding may not be a big deal to them, but I'd be cautious about making that decision, because on a page with lots of info like this I think it would be a useful tool (especially as I believe `hide` uses `display: none` so is accessible automatically?)

Reply:
this mirrors https://github.com/wagtail/wagtail/blob/master/wagtail/admin/static_src/wagtailadmin/js/page-editor.js#L238

which reminds me, I should probably sort out #6187 as it is relevant

Reply:
Haha, sorry, thought it was just a typo.

Yep, if it mirrors the other, I think that's fine to do separately. Maybe add an issue just so it doesn't get lost?

Reply:
made a note on #6187

Reply:
ah, the collapse/expand icon is output via iconfont in a `:before` rule. That needs to change to an icon, so added as To-Do on #6107

Last but not least, the click happens on the whole title-wrapper which should be the focus target